Doc/library/types.rst

```diff
@@ -480,13 +480,20 @@ Additional Utility Classes and Functions
    namespace, as well as a meaningful repr.
 
    Unlike :class:`object`, with ``SimpleNamespace`` you can add and remove
-   attributes.  If a ``SimpleNamespace`` object is initialized with keyword
+   attributes.
+   The :class:`!SimpleNamespace` constructor can take a positional argument
+   which must be a mapping object with string keys or an :term:`iterable`
+   object producing key-value pairs with string keys.
+   Key-value pairs from the mapping object or the iterable object are
+   directly added to the underlying namespace.
+   If a ``SimpleNamespace`` object is initialized with keyword
    arguments, those are directly added to the underlying namespace.
```

Comment:
Should the kwarg behaviour come first, as the longer-established case? I also think initialisation/arguments should be split to a new paragraph, for readability.

Reply:
I've suggested a re-wording!